server/scripts/OldAppointmentTimeSystemToNewSystemDatabaseEvolution.php

```diff
@@ -0,0 +1,49 @@
+<?php
+// This script will only work for Matthew Meacham
+
+$root = realpath($_SERVER['DOCUMENT_ROOT']);
+require_once "$root/server/user.class.php";
+$USER = new User();
+if (!$USER->isLoggedIn() || $USER->getUserId() !== '1') {
+	header("Location: /unauthorized");
+	die();
+}
+
+require_once "$root/server/config.php";
+
+evolveDatabase();
+
+function evolveDatabase() {
+	GLOBAL $DB_CONN;
+
+	$alreadyEvolved = true; // TODO: Update when actually running
+	if ($alreadyEvolved) {
+		die('The evolution has already occurred');
+	}
+
+	$query = 'SELECT AppointmentTime.appointmentTimeId, Site.doesInternational, Site.isVirtual, AppointmentTime.appointmentTypeId
+		FROM AppointmentTime
+			JOIN Site ON Site.siteId = AppointmentTime.siteId';
+	
+	$stmt = $DB_CONN->prepare($query);
+	$stmt->execute();
+	$appointmentTimes = $stmt->fetchAll(PDO::FETCH_ASSOC);
+
+	// Iterate through all the AppointmentTimes as assign them an appointmentTypeId
+	$query = 'UPDATE AppointmentTime
+		SET appointmentTypeId = ?
+		WHERE appointmentTime.appointmentTimeId = ?';
+	$stmt = $DB_CONN->prepare($query);
+	foreach ($appointmentTimes as &$appointmentTime) {
+		$appointmentTypeId = 1; // Residential
+		if ($appointmentTime['doesInternational'] == true && $appointmentTime['isVirtual'] == true) {
+			$appointmentTypeId = 7; // Virtual-China, this is the easiest thing we can do--just assign all international appointments as china
```

Comment:
I was just lazy when I wrote this. This code just updates the existing AppointmentTime table to minimize the changes that had to be made. Technically, we should add rows for the other international appointment types (India, Treaty, and Non-Treaty)--but we should also then re-assign existing appointments to the proper appointment time, which felt like a lot of complexity to me so I just ignored it. They are all historical appointments, so I didn't think it really mattered (it would just screw up data metrics)

Reply:
That makes sense! I was going to ask on our call about reassigning times and numbers of appointments, but that makes sense. So we're just updating the appointmentTypeId because it's not null?